docs/experimental-features/user-risk-score.asciidoc

```diff
@@ -0,0 +1,74 @@
+[[user-risk-score]]
+== User risk score
+
+NOTE: This feature is available for {stack} versions 8.3.0 and newer.
+
+The user risk score feature highlights risky usernames from within your environment. It utilizes a transform with a scripted metric aggregation to calculate user risk scores based on alerts  generated within the past 90 days. The transform runs hourly to update the score as new alerts are generated.
```

Comment:
A few suggested edits for conciseness:
```suggestion
The user risk score feature highlights risky usernames in your environment. It uses a transform with a scripted metric aggregation to calculate user risk scores based on alerts generated within the past 90 days. The transform runs hourly to update the score as new alerts are generated.
```

Reply:
Should score be plural in the last sentence? The second to last one has the plural version. 
```suggestion
The user risk score feature highlights risky usernames from within your environment. It utilizes a transform with a scripted metric aggregation to calculate user risk scores based on alerts  generated within the past 90 days. The transform runs hourly to update scores as new alerts are generated.
```